mne/decoding/time_gen.py

```diff
@@ -347,41 +410,34 @@ def score(self, epochs=None, y=None):
                                  'for scoring.')
         elif isinstance(y, list):
             y = np.array(y)
+
+        # Clean attributes
+        for att in ['scores_', 'y_true_']:
+            if hasattr(self, att):
+                delattr(self, att)
```

Comment:
They are accessible to the users in the same way as sklearn. Alex had asked me to delete them at the beginning of each call to ensure that we're not mixing along the way.